File: ScpProject/BL/BusinessObjects/Survey/Survey.cs
using System.Collections.Generic;
using b = BL.BusinessObjects;

namespace BL.BusinessObjects.Survey
{
    public class Survey
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int CreatedUserId { get; set; }
        public bool CanModify { get; set; }
        public bool IsDeleted { get; set; }
        public List<b.Tag> Tags { get; set; }
    }
}

File: ScpProject/StrengthConApp/src/app/Models/Survey.ts
import { Question } from './Question';
import { Tag } from './Tag';


export class Survey {
    public Id: number;
    public Name: string;
    public Description: string;
    public Questions: Question[] = [];
    public Tags: Tag[] = [];
    public WeekIds: number[] = [];
    public CanModify: boolean = true;
}